pkg/kube/apisix/apis/config/v2beta2/types.go

```diff
@@ -195,3 +195,40 @@ type ApisixRouteList struct {
 	metav1.ListMeta `json:"metadata" yaml:"metadata"`
 	Items           []ApisixRoute `json:"items,omitempty" yaml:"items,omitempty"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+
+// ApisixPluginConfig is the Schema for the ApisixPluginConfig resource.
+// An ApisixPluginConfig is used to support a group of plugin configs
+type ApisixPluginConfig struct {
+	metav1.TypeMeta   `json:",inline" yaml:",inline"`
+	metav1.ObjectMeta `json:"metadata" yaml:"metadata"`
+
+	// Spec defines the desired state of ApisixPluginConfigSpec.
+	Spec   ApisixPluginConfigSpec `json:"spec" yaml:"spec"`
+	Status v2alpha1.ApisixStatus  `json:"status,omitempty" yaml:"status,omitempty"`
+}
+
+// ApisixPluginConfigSpec defines the desired state of ApisixPluginConfigSpec.
+type ApisixPluginConfigSpec struct {
+	// +kubebuilder:validation:MinLength=1
+	Desc string `json:"desc,omitempty" yaml:"desc,omitempty"`
```

Comment:
Is this necessary? The `Desc` for the underlying `plugin_config` object is used for the management purpose, I think it's not useful for users, they can add descriptions from the annotations.

Reply:
I think we can keep it like this

> Is this necessary? The `Desc` for the underlying `plugin_config` object is used for the management purpose, I think it's not useful for users, they can add descriptions from the annotations.

I think we can keep it like this.

Reply:
But we don't have such field for other objects, and so far no users report such demands, so I doubt the necessity of it.

Reply:
If we look at it from the point of view of consistency with other resources, we really don’t need to keep it.

@neverCase could you please remove this field?